Your_first_ASDF_converter.ipynb

```diff
@@ -0,0 +1,554 @@
+    "package as the converters. For simplicity, they are for this\n",
+    "tutorial.\n",
+    "\n",
+    "We will name the converter package mfconverter (for My First\n",
+    "Converter) so this will create a package of that name in\n",
+    "the current directory. This tutorial will have the bare bones\n",
+    "files needed for that. The following will generate\n",
+    "such a barebones directory structure.\n",
+    "\n",
+    "Since this tutorial effectively is editing files and needs\n",
+    "to move between different directories, it is more awkward than\n",
+    "usual for a notebook\n",
+    "\n",
+    "Required Software installed\n",
+    "--------------------------------\n",
+    "\n",
+    "numpy\n",
+    "asdf v2.8 or higher\n",
+    "\n",
+    "Let's begin\n",
+    "-------------\n",
+    "\n",
+    "This will create a converter_tutorial directory in your home directory.\n",
+    "If you wish it to be elsewhere, make the appropriate changes to the\n",
+    "dependent commands, but if there is no strong reason to change it,\n",
+    "leave it be."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88e1f649",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir ~/converter_tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b1e163f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd ~/converter_tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06a42933",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Record current directory for restarts\n",
+    "import os\n",
+    "curdir = os.getcwd()\n",
+    "print(curdir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ae4a184",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir mfconverter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46d22a46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd mfconverter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a980c3f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8dbb76d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src/mfconverter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ede348bd",
+   "metadata": {},
+   "source": [
+    "And now we will create a module that has a very, very simple photo ID class and add a package `__init__.py`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f045f111",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/mfconverter/photo_id.py\n",
+    "class PhotoID:\n",
+    "    \"Holds Phot ID information\"\n",
+    "\n",
+    "    def __init__(self, last_name, first_name, image):\n",
```

Comment:
Any reason not to also name the last argument `photo`?

Reply:
Not really. I bow to photo.